app/stores/useLeaderboardStore.ts

```diff
@@ -0,0 +1,34 @@
+import { create } from "zustand"
+
+import {  Player } from "@interfaces"
+import { customFetch } from "@utils"
+
+interface LeaderboardState {
+    rankingFilter: string
+    allPlayers: Player[]
+    setRankingFilter:(filter:string) => void
+    fetchLeaderboard: (filter:string) => void
+}
+
+const useLeaderboardStore = create<LeaderboardState>((set) => ({
+    rankingFilter:'wins',
```

Comment:
Yeah, seems you have a default set here. Check back in the `Controls.tsx` then to determine why the filter display value won't change.